### PR TITLE
Added documentation about calling v3

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: docs/calling/calling_v3.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+1. Check for existing call
+1. Create an `ECall` entity with "state" `outgoing`
+1. Get user media stream
+1. Create local peer connection
+1. Create and set local SDP (type: offer)
+1. Initialized WebRTC data channel
+1. Generate ICE candidates (until last candidate or specified timeout)
+1. Create an `ECallMessage` of "type" `SETUP` and "response" set to `false`
+1. Wrap contents of `ECallMessage` in a generic message of type `calling`
+1. Send generic message to Bob via encrypted message channel (Wire backend)
```
Comment (lipis) [inline docs/calling/calling_v3.md:22]: 1, 2, 3, 4, ....

File: docs/calling/calling_v3.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+## Calling v3
+
+### Call Setup
+
+#### Alice calls Bob
+
+- Caller: Alice
+- Callee: Bob
+
+**Act 1: Alice**
```
Comment (lipis) [inline docs/calling/calling_v3.md:10]: maybe `##### Act 1: Alice` instead of bold..

File: docs/calling/calling_v3.md (hunk 1 above)
Comment (bennycode) [reply docs/calling/calling_v3.md:22]: 5, 6, 7, 8, ...

Comment (lipis) [reply docs/calling/calling_v3.md:22]: 9, 10, 11, 12..

**Pull request merged**